### PR TITLE
Vendor in `django-taggit-serializer`

### PR DESCRIPTION
**Pull request by rtpg**
This is after attempting to contact the original repo owner and not getting any replies.

The licenses are identitcal, so vendoring shouldn't be a problem on that front. 

I also moved this into `taggit.serializers` so that people can transition smoothly, or at their own pace/never.

### CODE REVIEW

**Comment by rtpg** [comment]
Alright this lowers overall project coverage, but this was mainly about vendoring in the existing project so I'm going to merge this in as-is. 

I plan on doing some cleanup for the serializers at a later point (as I think the current mechanism is more frail than it needs to be given DRF's capabilities), but I'll pull this in and just prepare a release.

File: taggit/serializers.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
+"""
+Django-taggit serializer support
+
+Originally vendored from https://github.com/glemmaPaul/django-taggit-serializer
+"""
+import json
+
+# Third party
+from django.utils.translation import ugettext_lazy as _
```
Comment (auvipy) [inline taggit/serializers.py:9]: ugettext_lazy could bee gettext_lazy

File: taggit/serializers.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+from rest_framework import serializers
+
+
+class TagList(list):
+    def __init__(self, *args, **kwargs):
+        pretty_print = kwargs.pop("pretty_print", True)
+        list.__init__(self, *args, **kwargs)
```
Comment (auvipy) [inline taggit/serializers.py:16]: what about using super call?

**Review by auvipy — COMMENTED** [review]
good job

**Pull request merged**